src/main/java/uk/gov/companieshouse/authcodenotification/service/PrivateDataRetrievalService.java

```diff
@@ -16,26 +16,26 @@
 @Service
 public class PrivateDataRetrievalService {
 
-    private static final String OVERSEAS_ENTITY_URI_SECTION = "/overseas-entity/%s/entity-data";
+    private static final String OVERSEAS_ENTITY_URI_SECTION = "/company/%s/registered-email-address";
 
     @Autowired
     private ApiClientService apiClientService;
 
-    public OverseasEntityDataApi getOverseasEntityData(String requestId, String companyNumber)
+    public RegisteredEmailAddressJson getRegisteredEmailAddressData(String requestId, String companyNumber)
```

Comment:
Would name this simply `getRegisteredEmailAddress`.

Reply:
Renamed.